stdlib/Random/src/RNGs.jl

```diff
@@ -141,27 +141,39 @@ hash(r::MersenneTwister, h::UInt) =
     foldr(hash, (r.seed, r.state, r.vals, r.ints, r.idxF, r.idxI); init=h)
 
 function show(io::IO, rng::MersenneTwister)
+    sep() = print(io, ", ")
+    sho(x) = show(io, x)
```

Comment:
I think this is over-prioritizing terse-ness and hinders readability

Please spell out these function invocations by hand at the call-sites